ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -24,8 +30,8 @@ time, mark, audio, video {
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
-  font-family: $kuiFontFamily;
+  font: inherit; /* 1 */
+  font-family: inherit; /* 1 */
```

Comment:
Yep, I hear you. What's your solution for where a comment applies to multiple lines? And for where multiple comments apply to a single line?